resources/ctia-default.properties

```diff
@@ -1,10 +1,10 @@
-auth.service.type=allow-all
+ctia.auth.type=allow-all
 ctia.http.port=3000
 ctia.http.min-threads=10
 ctia.http.max-threads=100
 ctia.nrepl.port=3001
 ctia.nrepl.enabled=true
-ctia.store.default=memory
```

Comment:
@gbuisson I like your approach.  I would like to get this change in, because compared to what we have now, it is an improvement, but I would like to make a new PR that implements your idea.